src/clj/game/cards/programs.clj

```diff
@@ -646,6 +646,16 @@
                                    :effect (effect (runner-install target {:no-cost true}))} card nil)
                                 (trash state side card)))}]}
 
+   "Plague"
+   {:prompt "Choose a server for Plague" :choices (req servers)
+    :msg (msg "target " target)
+    :req (req (not (:server-target card)))
+    :effect (effect (update! (assoc card :server-target target)))
```

Comment:
Based on a recent discussion in Slack, let's start saving "misc data" card values inside a `:special` key of the card state. So `assoc-in ... [:special :server-target]` and appropriate changes to the lines that read that value.